scripts/tool-install.sh

```diff
@@ -12,14 +12,20 @@ sudo apt-get install pfring nprobe ntopng ntopng-data n2disk cento nbox
 sudo apt-get install wireshark tshark
 sudo apt-get update
 
-#install bro
+#install dependencies for bro
 sudo apt-get install cmake make gcc g++ flex bison libpcap-dev libssl-dev python-dev swig zlib1g-dev
-sudo apt-get install bro
+
+#install C++ Actors Framework, Ignore the KEYEXPIRED 1503492954 or NO_PUBKEY error
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/libraries:/caf/xUbuntu_14.04/ /' > /etc/apt/sources.list.d/caf.list"
+sudo apt-get update
+sudo apt-get install caf
+
+#install bro
 sudo wget https://www.bro.org/downloads/bro-2.5.1.tar.gz
 mkdir /opt/bro
 sudo tar -xvzf bro-2.5.1.tar.gz -C /opt/bro
```

Comment:
The downloaded tar.gz file does not the required bin/ sub-directory. Please confirm the working of the bro binary. If the required binary is not available, we may need to build from sources.